kibana-reports/server/routes/utils/dataReportHelpers.ts

```diff
@@ -172,6 +172,11 @@ export const getEsData = (arrayHits, report) => {
       } else {
         hits.push(data);
       }
+
+      // Truncate to expected limit size
+      if (hits.length >= limit) {
```

Comment:
Is the `>` symbol necessary? If `hits.length == limit` then won't this block always return `hits` before the length can be greater than limit?

Reply:
Yes, it is supposed to be like that. In my opinion, this is safer and clearer. So I'd like to keep `>` as it is. Thanks!